packages/ts-example-jest-commonjs/jest.config.cjs

```
@@ -0,0 +1,23 @@
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/** @type {import('ts-jest').JestConfigWithTsJest} **/
+module.exports = {
+  testEnvironment: 'jsdom', // need to access to the browser objects
+  transform: {
+    '^.+\.tsx?$': ['ts-jest', {}],
```

ESLint

[error] Unnecessary escape character: \..

(no-useless-escape)